app/models/post.rb

```diff
@@ -0,0 +1,22 @@
+class Post < ApplicationRecord
+    validates :title, presence: true
+    validates :content, presence: true
+    scope :published, -> { where(published: true) }
+    scope :unpublished, -> { where(published: false) }
+
+    def published?
+      published
```

Comment:
where we are using this method ?, if not te please remove it

Reply:
remode

Reply:
removed